trin-beacon/src/test_utils.rs

```diff
@@ -51,33 +51,33 @@ pub fn get_light_client_update(number: u8) -> ForkVersionedLightClientUpdate {
     }
 }
 
-// Valid number range for the test cases is 0..4
+// Valid number range for the test cases is 0..1
 pub fn get_light_client_finality_update(number: u8) -> ForkVersionedLightClientFinalityUpdate {
     let lc_finality_update = std::fs::read(format!(
-        "../test_assets/beacon/capella/LightClientFinalityUpdate/ssz_random/case_{number}/serialized.ssz_snappy"
+        "../test_assets/beacon/deneb/LightClientFinalityUpdate/ssz_random/case_{number}/serialized.ssz_snappy"
     ))
         .expect("cannot find test asset");
     let mut decoder = snap::raw::Decoder::new();
     let lc_finality_update = decoder.decompress_vec(&lc_finality_update).unwrap();
     let lc_finality_update =
-        LightClientFinalityUpdate::from_ssz_bytes(&lc_finality_update, ForkName::Capella).unwrap();
+        LightClientFinalityUpdate::from_ssz_bytes(&lc_finality_update, ForkName::Deneb).unwrap();
 
     ForkVersionedLightClientFinalityUpdate {
         fork_name: ForkName::Deneb,
         update: lc_finality_update,
     }
 }
 
-// Valid number range for the test cases is 0..4
+// Valid number range for the test cases is 0..1
```

Comment:
*Tiniest of nits*: this looks just like rust range syntax, but then it's off by one. So I guess it's slightly better to use something like:
```suggestion
// Valid number range for the test cases is 0..2
```

or 
```suggestion
// Valid number range for the test cases is 0..=1
```